### PR TITLE
FEV-1358: clean up firebase logging

### PR DESCRIPTION
**Pull request by smeds1**
I removed firebase logging events that I don't believe anyone is using. Ideally this change will significantly cut down the number of events we log per survey and speed up firebase queries.

### CODE REVIEW

File: FluStudy_us/src/util/fluResults.ts [hunk 1]
```diff
@@ -76,19 +76,6 @@ export async function logFluResult() {
   _previousPinkAnswer = pinkAnswer;
 }
 
-export async function logNumLines() {
-  const state = (await getStore()).getState();
-
-  const numLinesAnswer = getSelectedButton(state, NumLinesSeenConfig);
-  if (_previousNumLines && _previousNumLines !== numLinesAnswer) {
-    logFirebaseEvent(FunnelEvents.NUM_LINES_ANSWER_CHANGED, {
```
Comment (rightparen) [inline FluStudy_us/src/util/fluResults.ts]: Is this not interesting?  I thought we had an insight from this.

Comment (smeds1) [reply FluStudy_us/src/util/fluResults.ts]: @rightparen We don't have a high contrast image in Chills, so the user doesn't answer the number of lines question anymore.

File: FluStudy_us/src/ui/ConnectedRootContainer.tsx [hunk 2]
```diff
@@ -282,8 +282,6 @@ class ConnectedRootContainer extends React.Component<Props & WithNamespaces> {
     switch (action.type) {
       case NavigationActions.NAVIGATE:
       case StackActions.PUSH:
-        return NavEvents.FORWARD;
-
```
[the remark this answers is not in the document]
Comment (smeds1) [reply FluStudy_us/src/ui/ConnectedRootContainer.tsx]: @tpaik18 I find it interesting when a user navigates backward. Knowing that they want to revisit content seems useful. Seeing that a user navigates forward doesn't really provide any insight. 

I think you're right that I didn't properly remove FORWARD from the switch statement. Thanks for pointing that out.

File: FluStudy_us/src/ui/ConnectedRootContainer.tsx [hunk 3]
```diff
@@ -282,8 +282,7 @@ class ConnectedRootContainer extends React.Component<Props & WithNamespaces> {
...
+        break;
```
Comment (audererob) [inline FluStudy_us/src/ui/ConnectedRootContainer.tsx]: If those two events are going to no-op (i.e. return undefined), then you can just remove the two cases in the switch entirely.

File: FluStudy_us/src/util/fluResults.ts [hunk 4]
```diff
@@ -76,19 +76,6 @@ export async function logFluResult() {
...
-      old_answer: _previousNumLines,
-      new_answer: numLinesAnswer,
-    });
-  }
-  _previousNumLines = numLinesAnswer;
-}
-
```
Comment (audererob) [inline FluStudy_us/src/util/fluResults.ts]: You can also remove the declaration of _previousNumLines at the top of the file.

File: FluStudy_us/src/ui/ConnectedRootContainer.tsx (hunk 3 above)
Comment (smeds1) [reply FluStudy_us/src/ui/ConnectedRootContainer.tsx]: Thanks, @audererob

**Pull request merged**